### DISCUSSION TITLE
some folders/files aren't visible but they exist

### DISCUSSION THREAD

**Discussion by nehal-96** [codesandbox/codesandbox-client#5277] [Q&A]
I am working on testing in React app I  can't see **coverage folder** created by jest library,  it takes me so long to find out it does exist but not visible when I used **ls** command in the terminal : 
**coverage  node_modules  package.json  public  src  yarn.lock**
![image](https://user-images.githubusercontent.com/63882004/102024201-40b7a580-3d99-11eb-9c13-b47310dba04d.png)
so why I can't access this kind of folder??

**Comment by SaraVieira** [comment]
Hey!

Can you please send us a sandbox id we can take a look at?

**Reply by nehal-96** [reply]
where can I get that id??

**Reply by SaraVieira** [reply]
The sandbox link

**Reply by nehal-96** [reply]
https://codesandbox.io/s/netflixclone-ylzgr
